exporter/lokiexporter/README.md

```diff
@@ -17,7 +17,201 @@ Exports data via HTTP to [Loki](https://grafana.com/docs/loki/latest/).
 
 This component is **deprecated**: Loki now supports native [OTLP ingestion](https://grafana.com/docs/loki/latest/send-data/otel/) starting from v3. Grafana Cloud also supports [OTLP native ingestion](https://grafana.com/docs/grafana-cloud/send-data/otlp/send-data-otlp) for logs. This component will be removed in September 2024.
```

Comment:
From this section of the README the plan is to remove this entire exporter in the next month. Why do we want to update the README for a week or two if we're just going to delete this entire component one release later (at most)? Just want to make sure I understand what's happening here, and the best way to communicate this to users. 👍

Reply:
Great catch @crobert-1 ,
As discussed with @jpkrohling , we are postponing of 2 months the removal of the Loki Exporter from the OTel Collector Contrib to give users more time to migrate to the new native OTLP endpoint.
I updated this PR to change "September" to "November".